sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -249,6 +249,9 @@ public void close() {
             ((Closeable) integration).close();
           }
         }
+        if (!options.getExecutorService().isShutdown()) {
+          options.getExecutorService().shutdown();
+        }
```

Comment:
`shutdown` only stops accepting new tasks, so it won't be a problem, otherwise, we'd need to call `shutdownNow` and wait for X period of time which is thread-safe, I believe this would be part of the `flush` method, but not on this one.